functions.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+
+/**
+ * connects SQL database to PHP
+ */
+function connectDB()
+{
+    $db = new PDO ('mysql:host=db; dbname=AlexCollection', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+/**
+ * pulls set names from PDO
+ *
+ * @param array $db the PDO connection to SQL
+ *
+ * @return string the data from SQL
+ */
+function pullSetData(PDO $db) :array
```

Comment:
naming could be clearer eg: retrieveCardSets, try to stay away from generic words like Data where poss